lib/restforce/error_code.rb

```diff
@@ -399,6 +399,8 @@ class WeblinkUrlInvalid < ResponseError; end
 
     class WrongControllerType < ResponseError; end
 
+    class BigObjectUnsupportedOperation < ResponseError; end
```

Comment:
Would be better to sort it alphabetically

Reply:
Updated @michaelwnyc